keywords/TestServeriOS.py

```diff
@@ -327,7 +327,7 @@ def stop(self):
         """
         1. Flush and close the logfile capturing the LiteServ output
         """
-        self.close_app()
+        # self.close_app()
```

Comment:
wanna to remove this line instead of commenting out? use for debug? if so, you can keep it